### PR TITLE
update presentation planning bullet points & link in week 1 schedule

### LINKED ISSUE #404: change instructions for planning presentations & add link in week schedule
Link to [presentation planning](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/weekly-projects.md#structure) in week 1 schedule

FAC10 gave positive (verbal) reviews of having this outline & [FACN1 enjoyed this](https://github.com/foundersandcoders/nazareth-curriculum/blob/master/stop-go-continue/fac-n1/week-13.md#user-testing-at-the-beginning-of-the-week) when it was introduced in week 13 & asked for it to be introduced earlier
> + I thought the bullet points for presentations were really good.
> + We've really come a long way. Bullet pointed queues might have been really useful along the way

But students rarely knew what to say during
> Show your use of Github

This might have been better for mentors just to highlight during Q&A time

### PR DESCRIPTION
**Pull request by jsms90**
+ Remove "Show your use of Github" - better for mentors just to highlight examples of good practice during Q&A time
+ update presentation planning bullet points
+ add link from week 1 schedule

fixes #404

### CODE REVIEW

**Comment by eliascodes** [comment]
@jsms90 Did you remove the git/PR history thing deliberately?

File: coursebook/general/weekly-projects.md [hunk 1]
```diff
@@ -41,16 +41,11 @@ Each team then has 15 minutes to present, with 5 minutes for questions. We will
...
-2. Show your use of Github
-  + commit history
-  + PR history
-3. Walk us through the UX (user experience) i.e. demo your website
-  + how well does it conform to your brief - the user stories
-4. Walk us through your code:
-  + talk us through your chosen file structure
-  + code itself: prioritise the things you learned that were new to you, which the rest of your cohort could benefit from hearing about:
-    + something you're proud of, that was really cool
+2. Walk us through the UX (user experience) i.e. demo your website
```
Comment (des-des) [inline coursebook/general/weekly-projects.md]: Yeah I think commit history is important.

Also, `walk us through` -> `walk the class through`?

Comment (des-des) [inline coursebook/general/weekly-projects.md]: Hmm I think this is confusing.

Walking through the ux might be walk through wire frames.

I would just say, walk the class though the website and its functionality or something like that

**Comment by jsms90** [comment]
@eliascodes in the issue
> But students rarely knew what to say during
>
> > Show your use of Github
> This might have been better for mentors just to highlight during Q&A time

I don't think _they_ can talk about their commit history & therefore plan it into the presentation time.

The way I did it the first time was also really bad. I was basically pulling them up on all the things they did wrong. Putting people on the spot like that while they're presenting and telling them how shit everything is was just really bad.

Instead what I did with FAC11 was go through each team's history before they presented and found a couple of good examples of descriptive commit messages, commits linked to issues, descriptive issue and PR titles, verbose body, etc. Positive reinforcement is a way nicer way of going about it.

But that kind of thing should probably be in a mentor guide #365

**Comment by eliascodes** [comment]
@jsms90 That sounds OK. Would have been useful to have something like that in the PR description 😉

File: coursebook/general/weekly-projects.md (hunk 1 above)
Comment (jsms90) [inline coursebook/general/weekly-projects.md]: :thinking: How about 
> Walk the class through a user's experience of your website

**Comment by jsms90** [comment]
@eliascodes updated the top comment

**Comment by des-des** [comment]
@jsms90 FYI, I get to pulls assigned to me fastest, so i am more likely to get to something faster if you re-assign me after changes

**Comment by jsms90** [comment]
@des-des Would you be able to help with #369 ? I feel like you have a much stronger handle on how to use github's navbar effectively

**Pull request merged**